pkg/rotator/rotator.go

```diff
@@ -45,14 +45,17 @@ const (
 
 var crLog = logf.Log.WithName("cert-rotation")
 
-var vwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"}
-
 var crdGVK = schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"}
 
 var _ manager.Runnable = &CertRotator{}
 
 var restartOnSecretRefresh = false
 
+type WebhookInfo struct {
+	Name types.NamespacedName
```

Comment:
Since all webhooks are cluster-scoped, why not just make this a string? It would get rid of a bunch of `webhook.Name.Name` in the code. You could add a method on `WebhookInfo` called `NamespacedName()` as syntactic sugar if you like to create the namespaced name on-demand.

Reply:
Done